app/assets/stylesheets/admin/global/_base.sass

```diff
@@ -16,6 +21,18 @@ div.container
   span
     @extend %visually-hidden
 
+thead, h1, h2, h3, h4, h5, h6
+  color: $heading-color
+
+thead, h1, h2, h3
+  font-family: $heading-fonts
+
+h1, h2, h3, h4, h5, h6
```

Comment:
Any reason these are different to the bugfix in this file? https://github.com/thefrontiergroup/cciwa-web/blob/bugfix/fix-text-in-inputs/app/assets/stylesheets/admin/global/_base.sass